central/notifiers/namespace_properties.go

```diff
@@ -0,0 +1,14 @@
+package notifiers
```

Comment:
Assuming you will move this to pkg in the second phase of reactor yes?

Reply:
Yes